engines/stark/visual/text.cpp

```diff
@@ -144,4 +159,13 @@ void VisualText::resetTexture() {
 	createTexture();
 }
 
+bool VisualText::isBlank() {
+	for (auto c : _text) {
```

Comment:
We can't use c++11 features yet. That means no foreach and no auto :(

ResidualVM follows the same rules as ScummVM, and we don't do c++11 there either. This is mostly because the CI server has not yet been updated with c++11 compatible compilers for all platforms.

Reply:
I see that the ```override``` specifier is used heavily so I thought c++11 is available. Looks like the c++11-compat.h redefines it. Neat.